expression/builtin_string_vec_test.go

```diff
@@ -34,7 +34,9 @@ var vecBuiltinStringCases = map[string][]vecExprBenchCase{
 	ast.Unhex: {
 		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETString}, geners: []dataGenerator{&randHexStrGener{10, 100}}},
 	},
-	ast.Trim:      {},
+	ast.Trim: {
+		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETString}},
```

Comment:
The `defaultGener` will never generate a string with leading or trailing space. We may need to add a new data generator or refine the `defaultGener`.

Reply:
I add a new data generator which called "randSpaceStrGener"